### PR TITLE
CNV CI - adding ci operator config files

### PR DESCRIPTION
**Pull request by lmilbaum**

### CODE REVIEW

File: ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
```
Comment (petr-muller) [inline ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml]: `cluster` is not necessary and creates a pitfall, better remove it

Comment (lioramilbaum) [reply ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml]: ack

**Comment by droslean** [comment]
/hold 

In case you want to wait for the rehearsal

**Comment by lioramilbaum** [comment]
@droslean I don't mind the rehearsal jobs are failing

**Comment by droslean** [comment]
@lioramilbaum You are able to cancel hold and merge it. For the next time :)

/hold cancel

**Pull request merged**